apps/web/src/data/en/index.ts

```diff
@@ -789,6 +790,8 @@ export const loggedInData = {
           altPlaceholder: 'Describe what the image shows',
           maxWidth: 'Maximum width',
           maxWidthPlaceholder: 'Enter the maximum width',
+          helpTooltipText: 'More information and help related to Image Plugin',
```

Comment:
Maybe a question for @GregorZupan , but what is the point of showing this text, when hovering on the question mark icon? I would expect either:
- The question mark icon to be clickable, so it shows more information and help in a modal
- This text to show the information and help, instead of this text

Reply:
Thanks for bringing it up!

When implementing the Epic (Help&Onboarding) the icon will be clickable and lead to the help section/tutorial for the plugin. For now we can use a more helpful hover tooltip (WIP)

Reply:
Hidden for now